cassdegrees/ui/views/staff/bulk_data_upload.py

```diff
@@ -182,22 +220,38 @@ def bulk_data_upload(request):
         # Stores the index of the column containing the data type of each row,
         # so that the right data is stored in the right column
         # This would also allow columns to be in any order, and courses/subplans would still be added.
-        map = {}
+        map = dict()
         for row in uploaded_file:
             if first_row_checked:
                 if content_type == 'Courses':
                     # If number of columns from file doesn't match the model, return error to user.
-                    if len(row) != 6:
+                    if len(row) != 11:
```

Comment:
Is it possible to not hard code this one? Not exactly your problem, but yeah.

Reply:
Unfortunately no, this is one of the checks to make sure the file format is in a desirable format.